templates/zerver/help/include/go-to-organization-settings.md

```diff
@@ -0,0 +1,6 @@
+1. From your desktop, click on the **gear**
+   (<i class="icon-vector-cog"></i>) in the upper right corner.
+
+1. Select **Settings**.
+
+1. Select **Organization** from the top left corner of the settings screen.
```

Comment:
These last two steps can just be:
```
1. Select **Manage organization**.
```

I would also potentially rename this macro to go-to-manage-organization?